.gitignore

```diff
@@ -4,3 +4,5 @@
 /node_modules
 /phpunit.xml
 /.phpunit.cache
+
+.idea/
```

Comment:
This [should be in your global `.gitignore` file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/).